src/pages/ourwork.js

```diff
@@ -23,305 +16,51 @@ import {
   ContactFormSubmit,
   ButtonLink,
 } from "../components/ourWork/styles"
-import { Group, Work } from "../components/ourWork/Works"
+import { PopupText } from "react-calendly"
 import { Button } from "../components/shared/Button"
-
 import Hero from "../components/ourWork/assets/heroX2.png"
+import { graphql } from "gatsby"
+import SEO from "../components/seo"
+import { Group, Work } from "../components/ourWork/Works"
 
-import MoliBG from "../components/ourWork/assets/moli-bg.svg"
-
-import YourVoiceBG from "../components/ourWork/assets/your-voice-bg.svg"
+export default function OurWork({ data: { ourWork } }) {
+  let works = []
 
-import CodeToLearnBG from "../components/ourWork/assets/code-to-learn-bg.svg"
+  const sortedEdges = [...ourWork.edges].sort(
+    (a, b) => a.node.data.id - b.node.data.id
+  )
 
-import IICBG from "../components/ourWork/assets/iic-bg.svg"
+  for (const edge of sortedEdges) {
+    const data = edge.node.data
 
-import PelvicHealthBG from "../components/ourWork/assets/pelvic-health-bg.svg"
+    if (data.image === null || data.tile === null) {
+      continue
+    }
 
-import Commit2ActBG from "../components/ourWork/assets/commit2act-bg.svg"
+    let work = {
+      title: data.title,
+      image: data.image.localFiles[0].childImageSharp.fluid,
+      aspectRatio: data.image.localFiles[0].childImageSharp.fluid.aspectRatio,
+    }
 
-import ConnectedNorthBG from "../components/ourWork/assets/connected-north-bg.svg"
-import PassivBG from "../components/ourWork/assets/passiv-bg.svg"
+    if (data.link !== null) {
+      work["link"] = data.link
+    }
 
-import JoniBG from "../components/ourWork/assets/joni-bg.svg"
+    if (data.background !== null) {
+      work["background"] = data.background.localFiles[0].url
+    }
 
-import BloomingHouseBg from "../components/ourWork/assets/blooming-house-bg.svg"
+    works.push(work)
+  }
 
+  const worked_grouped_by_threes = serializeWorks(works)
```

Comment:
I have not worked in gatsby so take comments with a grain of salt but we mostly work with camelCase for variables.

Reply:
Good catch, I've updated it to use camelCase